lms/envs/aws.py

```diff
@@ -188,6 +188,8 @@
 EDXMKTG_LOGGED_IN_COOKIE_NAME = ENV_TOKENS.get('EDXMKTG_LOGGED_IN_COOKIE_NAME', EDXMKTG_LOGGED_IN_COOKIE_NAME)
 EDXMKTG_USER_INFO_COOKIE_NAME = ENV_TOKENS.get('EDXMKTG_USER_INFO_COOKIE_NAME', EDXMKTG_USER_INFO_COOKIE_NAME)
 
+LMS_ROOT_URL = ENV_TOKENS.get('LMS_ROOT_URL')
```

Comment:
Hi, I'm following this change for PR #12934, which currently [constructs this root URL from the `settings.HTTPS` flag and `settings.SITE_NAME`](https://github.com/edx/edx-platform/pull/12934/files#diff-8b9331446df1cde8a9ebf1dd15bb6094R107).  I'm hoping to use this new env variable instead.

Is there a corresponding configuration PR to add this variable to [lms_env_config](https://github.com/edx/configuration/blob/d714a81fae0ce2f98bfc34151dc6f2b300ce2b90/playbooks/roles/edxapp/defaults/main.yml#L975)?